core/src/test/java/kafka/test/MockController.java

```diff
@@ -196,27 +189,24 @@ private MockController(Collection<MockTopic> initialTopics) {
     }
 
     @Override
-    public CompletableFuture<Map<ConfigResource, ApiError>> incrementalAlterConfigs(
-            Map<ConfigResource, Map<String, Map.Entry<AlterConfigOp.OpType, String>>> configChanges,
-            boolean validateOnly) {
+    public CompletableFuture<Map<ConfigResource, ApiError>> incrementalAlterConfigs(Map<ConfigResource, Map<String, Map.Entry<AlterConfigOp.OpType, String>>> configChanges,
+                                                                                    boolean validateOnly) {
```

Comment:
IMO, this would be better readable as
```suggestion
    public CompletableFuture<Map<ConfigResource, ApiError>> incrementalAlterConfigs(Map<ConfigResource,    
                                                                                    Map<String, Map.Entry<AlterConfigOp.OpType, String>>> configChanges,
                                                                                    boolean validateOnly) {
```
(Note: GitHub suggestions make it actually less readable 😅. Each parameter should be on its own line (except the first) and be aligned after the `(` of the method.)